.circleci/ds005_outputs.txt

```diff
@@ -6,6 +6,8 @@ fmriprep/logs/CITATION.md
 fmriprep/logs/CITATION.tex
 fmriprep/sub-01
 fmriprep/sub-01/anat
+fmriprep/sub-01/anat/sub-01_desc-aparcaseg_dseg.nii.gz
+fmriprep/sub-01/anat/sub-01_desc-aseg_dseg.nii.gz
 fmriprep/sub-01/anat/sub-01_desc-brain_mask.nii.gz
 fmriprep/sub-01/anat/sub-01_desc-preproc_T1w.nii.gz
 fmriprep/sub-01/anat/sub-01_dseg.nii.gz
```

Comment:
Thought for another PR: Since we do sometimes have other dseg files, do we want to specify `desc-fast` or similar?

Reply:
That would be a good idea. I'd rather go with `desc-3tissues` and leave the provenance for the json file.